train.py

```diff
@@ -851,11 +856,23 @@ def train_one_epoch(
     model.train()
 
     end = time.time()
-    num_batches_per_epoch = len(loader)
-    last_idx = num_batches_per_epoch - 1
+    num_batches_per_epoch = len(loader) // args.iters_to_accum + (1 if len(loader) % args.iters_to_accum > 0 else 0)
+    last_idx = len(loader) - 1
```

Comment:
couldn't this be `(len(loader) + args.iter_to_accum -1) // args.iters_to_accum` or `math.ceil(len(loader) / args.iter_to_accum)`

Reply:
Thanks!
I'll change this line to first one as you mentioned.
Second one needs `import math`, which looks verbose